### PR TITLE
add plugin support apache EventMesh

### PR DESCRIPTION
**Pull request by heihei180**
Closes : [#9910](https://github.com/apache/skywalking/issues/9910)

- [x] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #9910 .
- [ ] Update the [`CHANGES` log](https://github.com/apache/skywalking-java/blob/main/CHANGES.md).

### CODE REVIEW

**Comment by wu-sheng** [comment]
Hi,thanks for the contribution. 
You are required to follow plugin dev doc to submit test scenarios to help us verifying the plugin in the CI.

File: apm-protocol/apm-network/src/main/java/org/apache/skywalking/apm/network/trace/component/ComponentsDefine.java [hunk 1]
```diff
@@ -231,4 +231,6 @@ public class ComponentsDefine {
 
     public static final OfficialComponent IMPALA_JDBC_DRIVER = new OfficialComponent(133, "Impala-jdbc-driver");
 
+    public static final OfficialComponent EVENT_MESH = new OfficialComponent(134, "EventMesh");
```
Comment (wu-sheng) [inline apm-protocol/apm-network/src/main/java/org/apache/skywalking/apm/network/trace/component/ComponentsDefine.java:234]: This ID should be registered in the main repository first.

**Comment by wu-sheng** [comment]
https://skywalking.apache.org/docs/main/next/en/setup/service-agent/virtual-mq/

And tags are required, and peer should be set at consumer and producer sides.

**Comment by pg-yang** [comment]
The current ui repo is  https://github.com/apache/skywalking-booster-ui

File: apm-protocol/apm-network/src/main/java/org/apache/skywalking/apm/network/trace/component/ComponentsDefine.java (hunk 1 above)
Comment (wu-sheng) [reply apm-protocol/apm-network/src/main/java/org/apache/skywalking/apm/network/trace/component/ComponentsDefine.java:234]: https://github.com/apache/skywalking/blob/6984bbc65a4c12009e095e376b5e6ffb3a90db59/oap-server/server-starter/src/main/resources/component-libraries.yml#L434-L436

134 had been registered as Impala server, this is why we required for pre-registration before agent plugin PR.

File: apm-sniffer/apm-sdk-plugin/eventmesh-1.x-plugin/pom.xml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>apm-sdk-plugin</artifactId>
+        <groupId>org.apache.skywalking</groupId>
+        <version>8.13.0-SNAPSHOT</version>
```
Comment (wu-sheng) [inline apm-sniffer/apm-sdk-plugin/eventmesh-1.x-plugin/pom.xml:8]: Please sync to the main branch, and you need to use `8.14.0-SNAPSHOT`.

**Comment by wu-sheng** [comment]
Marked as stale as no update for a while.

**Pull request closed without merging**